workflows/testnet/report.go

```diff
@@ -156,6 +168,132 @@ func (r *Report) addScreenshotsToMarkdown(md *markdown.Markdown) {
 	}
 }
 
+func (r *Report) addLoadTestResultsToMarkdown(md *markdown.Markdown) {
```

Comment:
there's probably a better way of generating these reports, i'll take a look at that whenever I have free time, all good for now